test/src/main/java/org/apache/accumulo/test/ShellServerIT.java

```diff
@@ -2115,9 +2099,8 @@ private void setupRealContextPath() throws IOException {
   private void setupFakeContextPath() throws IOException {
     // Copy the test iterators jar to tmp
     Path baseDir = new Path(System.getProperty("user.dir"));
-    Path targetDir = new Path(baseDir, "target");
-    Path classesDir = new Path(targetDir, "classes");
-    Path jarPath = new Path(classesDir, "ShellServerIT-iterators.jar");
+    Path jarPath = new Path(baseDir + "/src/main/resources/org/apache/accumulo/test",
+        "ShellServerIT-iterators.jar");
```

Comment:
Since resources are copied to `target/classes/`, you could also do:

```suggestion
    Path jarPath = new Path(baseDir + "/target/classes/org/apache/accumulo/test",
        "ShellServerIT-iterators.jar");
```

However, I'm wondering if this should be in a different directory entirely, instead of `src/main/resources/`, since it does not appear to be needed inside the test jar itself. The same is true for any others... if they're not strictly needed inside the test jar, it's probably fine to take them out of `src/main/resources` and put them in a directory like `src/main/testjars` that won't get bundled into the built jar.

Reply:
I found this in the accumulo-test pom:
`<descriptor>src/main/test-jars/Iterators.xml</descriptor>`

I tried putting them in that directory but looks like they aren't getting picked up.

Reply:
Looks like there is a maven assembly plugin execution create-iterator-test-jar, to create "TestJar". I guess this is used for TestJar-Iterators.jar in `ShellServerIT` and `ScannerContextIT`.